tools/cmd/slim-jim/cmd/root.go

```diff
@@ -0,0 +1,44 @@
+package cmd
```

Comment:
let's make this a top-level remove command with sub commands in the same way as validate clean etc